build/cluster-operator-ansible/playbooks/cluster-api-prep/files/cluster-api-template.yaml

```diff
@@ -381,10 +383,12 @@ parameters:
 - name: SERVING_KEY
 # location of container image
 - name: IMAGE
```

Comment:
Can we be more specific with this name here while we are changing things? I recommend `CLUSTER_API_IMAGE` to match with the label used in other files.